astronomy/eclipse_test.cpp

```diff
@@ -250,28 +214,29 @@ class EclipseTest : public testing::Test {
         current_time - bisection_interval,
         current_time + bisection_interval);
     EXPECT_EQ(Sign(actual_contact_time - current_time),
-              Sign(time_error)) << NAMED(time_error);
+              Sign(time_error)) << NAMED(actual_contact_time - current_time);
```

Comment:
As, discussed, it's pretty messy to check that the error has the right sign and the right order of magnitude in a single statement:
```
AnyOf(AllOf(Lt(2), Gt(1)), AllOf(Gt(-2), Lt(-1)))
```
The logging can be simplified, though.  Also changed the sign check to use `EXPECT_THAT`.